conf/messages.cy

```diff
@@ -334,6 +342,15 @@ enterContactDetails.error.email.length = Mae’n rhaid i’r cyfeiriad e-bost fo
 enterContactDetails.error.phoneNumber.length = Mae’n rhaid i’r rhif ffôn fod yn 20 o gymeriadau neu lai
 enterContactDetails.error.phoneNumber.minLength = Mae’n rhaid i’r rhif ffôn fod yn 10 rif neu fwy
 enterContactDetails.error.phoneNumber.invalid = Nodwch rif ffôn, fel 01632 960000, 07700 900000 neu +44 0808 157 0192
+enterContactDetails.details.link.text = Os nad ydych am i ni gysylltu â chi drwy e-bost
+enterContactDetails.tariff.email.cy = gwasanaethcymraeg@hmrc.gov.uk
+enterContactDetails.tariff.email = tariffclassification@hmrc.gov.uk
```

Comment:
Again, not entirely sure why we need to split this and if there are actually 2 emails that can be used.